lib/annotation_manager.rb

```diff
@@ -22,8 +22,17 @@ def create_annotation_xml
     prompt_for_negation "Would you like to update and overwrite the annotations' TEI for reference in #{$annotation_file}?"
 
     create_annotations if !@flask_queried_bool
+
+    # Remove "Needs Correction" annotations
+    annotations = []
+    @flask_annotations.each do |anno|
+      if ! anno.tags.include? "Needs Correction"
```

Comment:
Maybe this config line is no longer being used to evaluate if the annotations should go through, but I would recommend filtering this entire list out here, if not:

https://github.com/Willa-Cather-Archive/annotonia-converter/blob/master/config.demo.rb#L16